### ISSUE TITLE
Startup error similar to ErrorWindowAddon

### ISSUE THREAD

**Issue by neil-hughes-108** [FlowingCode/FilterableLayoutAddon#15]
hi there, i sometimes see this error when starting a webapp that includes this addon, i notice it's a similar error to one reported for 
https://github.com/FlowingCode/ErrorWindowAddon/issues/2

```
java.lang.IllegalStateException: Running project in development mode with no access to folder '/var/jenkins_home/workspace/ErrorWindow-14-addon/target/checkout'.
Build project in production mode instead, see https://vaadin.com/docs/v14/flow/production/tutorial-production-mode-basic.html
```

many thanks
Neil

**Comment by javier-godoy** [comment]
Confirmed. The release for v2.0.1 contains flow-build-info.json

**Comment by javier-godoy** [comment]
Thanks for reporting this issue. Fix has been released in version 2.0.2.
